app/controllers/sidekiq_monitoring/application_controller.rb

```diff
@@ -0,0 +1,5 @@
+module SidekiqMonitoring
+  class ApplicationController < ActionController::Base
+    protect_from_forgery with: :exception
+  end
```

Comment:
I'm not entiely sure, but devise just uses the `ApplicationController` of the current application. Maybe this would be a good idea too?

Reply:
I'll have a look into that!